docs/hub/api.md

```diff
@@ -46,12 +46,16 @@ params = {
 
 This is equivalent to `huggingface_hub.list_models()`.
 
+The key `createdAt` in the response begins on `2022-03-02T23:29:04.000Z` as a result of a special migration.
```

Comment:
IMO this should be mentioned just once, under a `# Repo creation times` heading, with a wording like:
> For all repos-returning API calls, the `createdAt` attribute is the time when the repo was created.
> Note that there is a special value of `2022-03-02T23:29:04.000Z` for all repos that were created before we started storing the creation date.

cc'ing @Wauplin too

Reply:
Yeah sure